pkgs/applications/graphics/brlcad/default.nix

```diff
@@ -36,9 +38,21 @@ stdenv.mkDerivation rec {
     "-DBRLCAD_ENABLE_STRICT=OFF"
   ];
 
+  env.NIX_CFLAGS_COMPILE = toString [
+    # Needed with GCC 12
+    "-Wno-error=array-bounds"
+  ];
+
+  postInstall = ''
+    echo "----------------------------------------------------"
+    patchelf --print-rpath "$out/bin/tclsh"
```

Comment:
```
----------------------------------------------------
/build/source/build/lib::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::/nix/store/rq2hjvfgq2nvh5zxch51ij34rqqdpark-brlcad-7.38.0/lib:/nix/store/4znq11s8j9d29kj2l4qivl5pyhbbzy8q-zlib-1.3/lib:/nix/store/7jiqcrg061xi5clniy7z5pvkc4jiaqav-glibc-2.38-27/lib:/nix/store/np3cndfk53miqg2cilv7vfdxckga665h-gcc-13.2.0-lib/lib
----------------------------------------------------
```

not sure what is there

Reply:
After a never ending `bisect`, I ended up finding the culprit:
https://github.com/BRL-CAD/brlcad/commit/fbdbf042b2db4c7d46839a17bbf4985cdb81f0ae
@SuperSandro2000 do you know what to do in this case ?